TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json

```diff
@@ -0,0 +1,71 @@
+// diagnostic_report.Z058.ct0433.json
+// id: Z058
```

Comment:
I have made it a patter to isolate the patient "name" from the "exam" name

The Breast Dx reports follow the same pattern



________________________________
From: Marc Kohli [mailto:notifications@github.com]
Sent: Thursday, February 19, 2015 12:05 PM
To: shadowdoc/siim-dataset
Cc: Langer, Steve G., Ph.D.
Subject: Re: [siim-dataset] Draft of json objects for patient Z058. (#6)


In TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json<https://github.com/shadowdoc/siim-dataset/pull/6#discussion_r25009672>:

> @@ -0,0 +1,71 @@
> +// diagnostic_report.Z058.ct0433.json
> +// id: Z058


this ID should be Z058ct0443

-
Reply to this email directly or view it on GitHub<https://github.com/shadowdoc/siim-dataset/pull/6/files#r25009672>.